<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1283</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colonia de bacterias</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuracion</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Largo:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinW">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ancho:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinH">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>81</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAceptar">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>110</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="boxAleatorio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Configuracion aleatoria</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinPBac">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>70</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prob. bacterias:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tamaño de punto:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinPunto">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>110</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>7</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tiempo/generacion:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBreed">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Energía/nutriente:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinEFood">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Energía/inicio:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinEStart">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prob.nutriente:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinPFood">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>70</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>70</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Prob. toxinas:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinPToxin">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>70</y>
       <width>66</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Energía min fisión:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinMinE">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Daño/toxina:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinDToxin">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Lapso/toxinas:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinLToxin">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>70</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>30</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Energía/iteración:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinEIt">
     <property name="geometry">
      <rect>
       <x>1200</x>
       <y>30</y>
       <width>47</width>
       <height>24</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableCells">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblGen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteracion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBacterias">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bacterias</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblToxins">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Toxinas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nutrientes</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFood">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
